README.md

```diff
@@ -69,6 +69,8 @@ Getting started
     The `SimpleCov.start` **must** be issued **before any of your application
     code is required!**
 
+    This is especially run if you run anything that keeps your tests application loaded like spring, check out the **[spring section](#want-to-use-spring-with-simplecov)**.
```

Comment:
`especially run`? I guess there should be other word?

Reply:
oh good catch :sweat_smile: